packages/orbit-components/src/ChoiceGroup/ChoiceGroup.stories.tsx

```diff
@@ -10,103 +11,94 @@ import ChoiceGroup from ".";
 
 ChoiceGroup.displayName = "ChoiceGroup";
 
-export default {
+type ChoiceGroupPropsAndCustomArgs = React.ComponentProps<typeof ChoiceGroup> & {
+  boxShadowSize: string;
+};
+
+const meta: Meta<ChoiceGroupPropsAndCustomArgs> = {
   title: "ChoiceGroup",
+  component: ChoiceGroup,
+
+  args: {
+    label: "What was the reason for your cancellation?",
+    onlySelectionText: "Only",
+    labelSize: LABEL_SIZES.NORMAL,
+    labelAs: LABEL_ELEMENTS.H4,
+    error: "",
+    filter: false,
+    onChange: action("onChange"),
+  },
+
+  argTypes: {
+    labelSize: {
+      options: Object.values(LABEL_SIZES),
+      control: {
+        type: "select",
+      },
+    },
+    labelAs: {
+      options: Object.values(LABEL_ELEMENTS),
+      control: {
+        type: "select",
+      },
+    },
+  },
 };
 
-export const Default = ({ label }) => {
-  return (
-    <ChoiceGroup label={label} onChange={action("onChange")}>
+export default meta;
+type Story = StoryObj<ChoiceGroupPropsAndCustomArgs>;
+
+export const Default: Story = {
+  render: args => (
+    <ChoiceGroup {...args}>
       <Radio label="Reason one" value="one" />
       <Radio label="Reason two" value="two" />
       <Radio label="Reason three" value="three" />
     </ChoiceGroup>
-  );
-};
-
-Default.story = {
-  parameters: {
-    info: "Playground of ChoiceGroup",
-  },
-};
-
-Default.args = {
-  label: "What was the reason for your cancellation?",
+  ),
 };
 
-export const Multiple = ({ label }) => {
-  return (
-    <ChoiceGroup label={label} onChange={action("onChange")}>
+export const Multiple: Story = {
```

Comment:
Hm, when I set `filter = true`, I still don't see `onlySelectionText`, same for WithError story. In the docs it's written it's for using `onOnlySelection` callback which is not in those stories, so maybe it can be excluded? Or callback added :D